### PR TITLE
routing useless routes to /Home and ESlint request to victory index/s…

### PR DESCRIPTION
**Pull request by FrancoisSalmon**
…tyle

### CODE REVIEW

**Review by camillaume — APPROVED** [review]
OK

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ function App() {
         <Header />
         <main>
           <Routes>
+            <Route path="*" element={<Home />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:20]: Hmm, I have my doubt about this.
That's the syntax I'd use for a 404 error, but it might work as intended

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**